typescript/src/tomas_toniut/StringCalculator.test.ts

```diff
@@ -52,5 +52,10 @@ describe(StringCalculator, () => {
     expect(result).toEqual(10);
   });
 
+  it("debe retornar Error, si la entrada son numeros negativos", () => {
+    const stringCalculator = new StringCalculator();
+    expect(() =>(stringCalculator.add('-2,-4'))).toThrow('negativos no están permitidos: -2,-4');
```

Comment:
Cuidado, acá no estás usando los datos de prueba provistos y estás haciendo un escenario simplificado.

Especificación: `-1,2,-3` debe lanzar excepción `negatives not allowed: -1,-3`